index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400|Roboto&display=swap" rel="stylesheet">  
+    <title>Apple- clone</title>
+</head>
+<body>
+    <div class="top container">
+        <header>
+            <nav>
+                <ul>
+                    <li><a href="#"> <img src="assets/Apple-Logo.png" alt="Apple"> </a></li>
+                    <span></span>
+                    <li><a href="#">Store</a></li>
+                    <span></span>
+                    <li><a href="#">Mac</a></li>
+                    <span></span>
+                    <li><a href="#">iPod</a></li>
+                    <span></span>
+                    <li><a href="#">iPhone</a></li>
+                    <span></span>
+                    <li><a href="#">iPad</a></li>
+                    <span></span>
+                    <li><a href="#">iTunes</a></li>
+                    <span></span>
+                    <li><a href="#">Support</a></li>
+                    <span></span>
+                    <li><input type="text"></li>
+                </ul>
+            </nav>
+            <a href="#"><h1>What will your verce be?</h1> <br>
+                Everyone has something to share. See how people are using <br>
+                iPad in their unique ways to add to the world's story.</a> 
+                <a href="link">Watch the "Your Verce" film and explore the stories <img src="assets/play-icon" alt="Play"></a>
+        </header>
+
+    </div>
+    <div class="sec container">
```

Comment:
For the sake of this project, the footer must start here. It will be divided into two parts: top, and bottom. The top will be this part, with the four link images. The bottom will be the one that is already inside the footer tag.